File: QolmsYappli/QolmsYappli/Models/JsonParameter/AuthTicketJsonParameter.vb
Imports System.Runtime.Serialization

<DataContract()>
<Serializable()>
Public NotInheritable Class AuthTicketJsonParameter
    Inherits QyJsonParameterBase

#Region "Public Property"

    ' TODO: 要検討
    <DataMember()>
    Public Property UserId As String = String.Empty

#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="AuthTicketJsonParameter" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()

        MyBase.New()

    End Sub

#End Region

End Class

File: QolmsJotoWebView/Models/AuthTicketJsonParameter.cs
using System;
using System.Runtime.Serialization;

namespace MGF.QOLMS.QolmsJotoWebView
{
    [DataContract()]
    [Serializable()]
    public class AuthTicketJsonParameter: QjJsonParameterBase
    {
        #region "Public Property"

        /// <summary>
        /// ユーザーID
        /// </summary>
        [DataMember()]
        public string UserId { get; set; } = string.Empty;

        #endregion

        #region "Constructor"
        public AuthTicketJsonParameter() : base() { }

        #endregion
    }
}